src/extension/background-script/actions/ln/transactions.ts

```diff
@@ -0,0 +1,45 @@
+import utils from "~/common/lib/utils";
+import state from "~/extension/background-script/state";
+import type { Invoice, MessageTransactions } from "~/types";
+
+const transactions = async (message: MessageTransactions) => {
+  const isSettled = message.args.isSettled;
```

Comment:
Are there any use cases for loading invoices that are not yet settled? I think we don't display them anywhere, do we?

Reply:
we don't load unsettled invoices that is why we do check. we can remove that. but we wait for multiple testings of endpoint so that we are sure, no unsettled invoices are returned by getTransactions

Reply:
you can still filter them out, right? maybe we just do not need isSettled in message.args?

Reply:
resolved